### DISCUSSION TITLE
Erreur accès ressources par le gestionnaire de ressource V4.3.4

### DISCUSSION THREAD

**Discussion by Kiki-zoro** [JeromeDevome/GRR#371] [General]
Bonjour, j'ai un souci avec la version GRR V4.3.4 les gestionnaires de ressources n'ont pas accès à la page de modification, sauf s'ils sont admin d'un domaine même si ce n'est pas celui associé à la ressource.
<img width="475" alt="Capture d'écran 2023-03-16 074021" src="https://github.com/JeromeDevome/GRR/assets/169878690/4bd9f327-77ad-421b-ad90-57c7d64ad137">

**Comment by taib27** [comment]
il faut regarder de coté **Configuration général** ensuit **Accès et droits** pour vérifier qui peut Supp/modifier une Résa

**Reply by Kiki-zoro** [reply]
Bonjour, j'ai regardé mais rien vu qui empêche la modification tout le monde accèdent bien à la fiche descriptions de la ressource mais c'est quand on essaye de modifier la description en cliquant sur la roue cranté que le message apparait sauf si l'utilisateur connecté est admin d'une ressource n'importe laquelle.

**Reply by taib27** [reply]
Si les paramètres sont ok a ce niveau il faut ensuite voir coté **utilisateurs et accès** 
sélectionnez le domaine , ressources et voir qui gère la ressource.
un user simple ne modifier la ressource .

**Comment by Kiki-zoro** [comment]
Les utilisateurs concernés on bien accès au ressource puisqu'il y a bien la roue crantée qui leur propose de modifier.
Avant le changement de version ça fonctionnait sans problème. Il faudra que j'essaye avec un base vierge si c'est un problème de migration de base

**Reply by Kiki-zoro** [reply]
Je viens de faire un test base vierge, j'attribue une ressource à un utilisateur dans utilisateurs et accès /  gestion des ressources. J'ai donc Liste des personnes autorisées à gérer la/les ressource(s) ci-contre (hormis les administrateurs) : le nom de mon usager.
Je me connecter avec cette usage, j'ai bien la roue cranté 
![image](https://github.com/JeromeDevome/GRR/assets/169878690/3ed690ed-f2ec-49df-bf49-6910aae5841a)
et quand je clique sur la roue 
![image](https://github.com/JeromeDevome/GRR/assets/169878690/73beac95-988e-46f3-8a53-ab737a011777)

Donc le problème es facilement reproduisible si vous pouvez tester !!
![image](https://github.com/JeromeDevome/GRR/assets/169878690/06050350-588d-48e0-852a-2cd4a8897aba)
Merci

**Comment by Kiki-zoro** [comment]
Personne n'a pu faire le test sur un serveur ?

**Comment by Kiki-zoro** [comment]
Bonjour, j'ai toujours un problème de ressources si je mets toto gestionnaire ressource A, et admin du domaine B (la ressource étant dans le domaine A).
toto à accès à la ressource et a la page de modification mais il a une erreur d'accès à l'enregistrement
si toto n'est pas admin du domaine B il n'a pas accès a la pas de modification.
si toto est admin du domaine A tout est OK, mais je ne veux pas que me gestionnaire de ressource soit admin du domaine.
Comment procéder ?

**Comment by Kiki-zoro** [comment]
Je viens de faire un test avec le site https://demo3.mygrr.net/
connexion Utilisateur2 utilisateur2
Selection sites VALABRE  / Domaines MET essayer de modifier la ressource, lors de l'enregistrement message accès refusé
![image](https://github.com/JeromeDevome/GRR/assets/169878690/394bb006-bafa-4cc1-bd92-ec1a90001754)

**Comment by ynaessens** [comment]
Bonjour,
c'était une régression  de la version 3.5.1 à la version 4.3.
A priori le commit https://github.com/JeromeDevome/GRR/commit/d6e5cba271df3216a7a6d8f857085e2ba9abfe8d corrige le problème.
Merci de le confirmer.
Cordialement,
YN
